src/pool/srv_layout.h

```diff
@@ -83,6 +83,14 @@ struct pool_hdl {
 	uint64_t	ph_flags;
 	uint64_t	ph_sec_capas;
 	char		ph_machine[MAXHOSTNAMELEN+1];
+	size_t		ph_cred_len;
+	char		ph_cred[];
+};
+
+/* old format (<= version 2.0) */
+struct pool_hdl_v20 {
```

Comment:
[Nit] It seems like we should put the global version, instead of the release version, after the "v". E.g., if we intend to land this patch to release/2.2 (global version == 1), then we should write "v0" here.

Reply:
Ok, understand!

Reply:
Thanks.